luigi/rnacentral/genome_mapping/genome_mapping.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+"""
+Copyright [2009-2018] EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import csv
+import subprocess
+import sys
+
+from rnacentral.psql import PsqlWrapper
+
+import itertools as it
+
+from Bio.Seq import Seq
+from Bio.SeqRecord import SeqRecord
+
+
+
+FASTA = """
+with ensembl_ids as (
```

Comment:
We should be more generic, as there are other databases besides Ensembl that we can trust. For example lncipedia provides us coordinates that we trust. I would suggest changing query to export everything that does not already have a coordinate. We can then decide what database we use for coordinates and not populate the coordinate table for other databases.

Reply:
I agree that this would be the right thing to do but we need to clean up the `rnc_coordinates` table first. I added this to an issue where I keep track of genome mapping improvements: https://github.com/RNAcentral/rnacentral-webcode/issues/145